.github/workflows/dev-container.yml

```diff
@@ -195,7 +195,7 @@ jobs:
           push: never
           # TODO-24: Add custom @dependabot-like job to open PR to update dependencies
           env: |
-            SKIP=poetry-export,poetry-lock
+            SKIP=poetry-export,poetry-lock,ggshield
```

Comment:
It really makes sense to run it on server side to detect problems in case of people do not run hooks locally.